dtfs-central-api/src/v1/controllers/cron-jobs/cron-jobs.controller.ts

```diff
@@ -29,6 +19,13 @@ export const deleteAllEstoreLogs = async (req: CustomExpressRequest<{ reqBody: {
 
     return res.status(200).send();
   } catch (error) {
+    if (error instanceof InvalidAuditDetailsError) {
```

Comment:
I think I disagree - having multiple try-catch blocks seems to be against the spirit of what a try-catch is. In this case specifically, the only reason it would be unclear that the error came from the `validateAuditDetails` function is that it is not immediately obvious that that function throws. I prefer to call functions which throw on error something like `assert<Thing>IsValid` or to write

```typescript
const validationError = validateAuditDetails(auditDetails);
if (validationError) {
  return res.status(HttpStatusCode.BadRequest).send(validationError);
}
```

However, in this case, we do have specific error handling around whether or not the error is linked to the audit details and I think that does make it obvious without the tightly coupled try-catch blocks. If we wanted the tightly coupled try-catch blocks as an option, I would prefer to go with the cope snippet above